frontend/src/components/Header.jsx

```diff
@@ -6,7 +6,9 @@ import "./Header.css";
 function Header() {
   return (
     <header className="header">
-      <img src={logo1} alt="Header" className="logo1" />
+      <a href="/">
```

Comment:
This is a link to an internal route of your app. Use `<Link />` of `react-router` instead of a `<a>` tag.
[Lien vers la doc](https://reactrouter.com/en/main/components/link)